libpod/container_internal.go

```diff
@@ -1866,15 +1867,35 @@ func (c *Container) writeStringToStaticDir(filename, contents string) (string, e
 }
 
 // appendStringToRundir appends the provided string to the runtimedir file
-func (c *Container) appendStringToRundir(destFile, output string) (string, error) {
+func (c *Container) appendStringToRundir(destFile, output string, checkExist bool) (string, error) {
```

Comment:
Not sure we need the checkExists parameter, why not do this by default?  Do you envision this function being used in the future where something wants to just keep appending on the end of a file?

Reply:
I will change it to default